icu4c/source/samples/Makefile.in

```diff
@@ -22,7 +22,8 @@ include @platform_make_fragment@
 ## Files to remove for 'make clean'
 CLEANFILES = *~
 
-SUBDIRS = date cal
+SUBDIRS = cal citer coll csdet date datecal ufortune uresb datefmt msgfmt numfmt props strsrch translit uciter8 ucnv udata ustring dtitvfmtsample dtptngsample plurfmtsample
+
 ALLSUBDIRS = break case csdet datefmt msgfmt numfmt props translit ucnv udata ufortune uresb ustring citer uciter8 ugrep
```

Comment:
Should these be in `ALLSUBDIRS` ? (Not sure)

Reply:
I see no reason for ALLSUBDIRS. I've removed this definition and use.